fe/fe-core/src/main/java/org/apache/doris/transaction/TransactionState.java

```diff
@@ -298,6 +299,16 @@ public SchemaInfo(OlapTable olapTable) {
     // table id -> schema info
     private Map<Long, SchemaInfo> txnSchemas = new HashMap<>();
 
+    @Getter
+    @Setter
+    private List<SubTransactionState> subTransactionStates;
+    @Getter
+    @SerializedName(value = "subTxnIdToTableCommitInfo")
```

Comment:
Use short name, such as:
```suggestion
    @SerializedName(value = "sti")
```